### ISSUE TITLE
[Firefox 70] Active Tab shows through AC Popup With 2 Lines Drop-Down

### ISSUE THREAD

**Issue by MarkRH** [Aris-t2/CustomCSSforFx#229]
1. Describe the issue
Testing with a clean profile and a userChrome.css of:
```css
/* Aris imports from https://github.com/aris-t2/customcssforfx */
/* version 2.7.0 ******************************************************************************/

@import "./classic270/config/general_variables.css"; /**/
/*@import "./classic270/config/color_variables.css"; /**/  /* <- 'classic grey' colors */
@import "./classic270/css/tabs/classic_squared_tabs.css"; /**/
@import "./classic270/css/tabs/tabs_below_navigation_toolbar_fx65.css"; /**/ 						/* <-- tabs not on top Fx65+ */
/* @import "./classic270/css/tabs/tabs_below_navigation_toolbar_fx68.css"; /**/ /* <-- tabs not on top - Fx68+ - Windows/Linux */
@import "./classic270/css/tabs/missing_tab_favicon_restored_default.css"; /**/
@import "./classic270/css/tabs/tab_close_always_visible.css"; /**/
@import "./classic270/css/tabs/tab_icon_colors.css"; /**/
/*@import "./classic270/css/locationbar/ac_popup_classic_with_two_lines_fx65.css"; /**/  /* <--- old autocomplete popup for Fx 65+ */
@import "./classic270/css/locationbar/ac_popup_classic_with_two_lines_fx68.css"; /**/  /* <--- old autocomplete popup for Fx 64+ */
```
It looks like: 
![image](https://user-images.githubusercontent.com/8035962/64228935-041ab000-ceae-11e9-80c8-37998a09573e.png)

2. What should have happened instead?
Active Tab shouldn't show through.

3. Steps to reproduce the issue?
Run 70 Beta with the above includes.

4. SCREENSHOTS with/without this projects files (drag & drop images into this post)
See above

5. System information
- OS & OS version:  Windows 7 Pro 64bit 
- OS theme name: Classic
- DPI / HiDPI resolution: Normal/100%
- Firefox version: 70.0b3 for this test (also happens in 71 Nightly)
- Firefox theme / lw-theme: Default
- This projects settings the issue occurs with: Tabs below navigation toolbar

**Comment by Aris-t2** [comment]
```css
.urlbarView:not(.megabar),
#urlbar-results {
  z-index: 100 !important;
}
```

Will be fixed on next update.

**Comment by Aris-t2** [comment]
https://github.com/Aris-t2/CustomCSSforFx/releases/

**Comment by MarkRH** [comment]
OK, that seems to work.  I did add an override to add a border back to the bottom of the tab that was present in the fx65 version.

```css
/* override part in css/tabs/tabs_below_navigation_toolbar_fx68.css */
#TabsToolbar,
#main-window:not([style*='--lwt-header-image']):-moz-lwtheme-brighttext #TabsToolbar:-moz-lwtheme, 
#main-window:not([style*='--lwt-header-image']):-moz-lwtheme:-moz-lwtheme-brighttext #TabsToolbar:-moz-lwtheme {
  border-bottom: 1px solid !important;
}
```

Thanks!

**Comment by Aris-t2** [comment]
Its some kind of weird play a Mozilla to add or remove the border there with Every Fx version. No idea whats wrong with them...

**Comment by Aris-t2** [comment]
Active tab issue fixed in 2.7.2 release.
